tests/testthat/test-Model-class.R

```diff
@@ -1094,6 +1094,122 @@ test_that("No NA is returned in dose calculations for OneParExpNormalPrior model
   expect_false(is.na(calc_dose(0.95)))
 })
 
+# OneParExpExpPrior ----
+
+## constructor ----
+
+test_that("OneParExpExpPrior object can be created with user constructor", {
+  result <- expect_silent(
+    OneParExpExpPrior(
+      skel_probs = seq(from = 0.1, to = 0.9, length = 5),
+      dose_grid = 1:5,
+      lambda = 2
+    )
+  )
+  expect_valid(result, "OneParExpExpPrior")
+  expect_identical(result@skel_probs, seq(from = 0.1, to = 0.9, length = 5))
+  expect_identical(result@lambda, 2)
```

Comment:
How about checking `skel_fun` and `skel_fun_inv` slots?

Reply:
I mean to check the content of both functions. Yes, it is necessarily, as we must ensure, that the user constructor works in a way that is expected, that is really creates valid interpolation based on the input (i.e. `skel_probs`, `dose_grid`).